### PR TITLE
Kinda mobile friendly room booking app

### PR DESCRIPTION
**Pull request by manuatarchilogic**
AL-2208

I had to revert the Dependabot node-fetch bump because the app would not start with node-fetch > 3. Should I look into it?

Main changes are in the last commit.

### CODE REVIEW

**Comment by manuatarchilogic** [comment]
Screen Shot 2022-05-17 at 11 50 44
Screen Shot 2022-05-17 at 11 51 28
Screen Shot 2022-05-17 at 11 51 09
Screen Shot 2022-05-17 at 11 50 56

**Review by bnjm — APPROVED** [review]
I ran it locally and it feels pretty good 🙏  thanks for putting the style changes in one commit

File: server/package.json [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "mdaguerre",
+  "license": "ISC",
+  "dependencies": {
+    "cors": "^2.8.5",
+    "dotenv": "^8.2.0",
+    "express": "^4.17.1",
+    "http-proxy-middleware": "^1.0.3",
+    "node-fetch": "^2.6.0"
```
Comment (frederic-schwarz) [inline server/package.json:16]: v2.6.7 is the safe and patched version if v3 is not an option

**Review by frederic-schwarz — APPROVED** [review]

File: src/components/TimeLine/TimeLine.tsx [hunk 2]
```diff
@@ -1,43 +1,65 @@
...
-        <>
-            <div className="busyTimeline">
-                {props.usedSlots.map((isBusy, index) => {
-                    const selectedSlotClass = (Number(props.selectedTimeSlot) + Number(props.bookingSlots)) > index && props.selectedTimeSlot <= index  ? 'selectedSlot': '';
-                    const busyClass = isBusy? 'busy' : 'notBusy';
-                    return <div key={index} className={`slot ${selectedSlotClass} ${busyClass} `}>&nbsp;</div>;
-                })
-                }
+  return (
+    <>
```
Comment (romandkos) [inline src/components/TimeLine/TimeLine.tsx:16]: Something weird here

File: src/components/BookForm/BookForm.tsx [hunk 3]
```diff
@@ -1,183 +1,192 @@
...
+      email: ''
+    }
+  }
+
+  // select by default the maximum bookable time
+  const [bookingSlotsNumber, setBookingSlotsNumber] = useState<number>(
+    existingBooking ? existingBooking.duration : maxBookingSize
+  )
+  return (
+    <>
```
Comment (romandkos) [inline src/components/BookForm/BookForm.tsx:112]: and here, but I'm not expert in React :)

**Review by romandkos — APPROVED** [review]

File: src/components/TimeLine/TimeLine.tsx (hunk 2 above)
Comment (manuatarchilogic) [reply src/components/TimeLine/TimeLine.tsx:16]: https://reactjs.org/docs/fragments.html#short-syntax

File: src/components/BookForm/BookForm.tsx (hunk 3 above)
Comment (manuatarchilogic) [reply src/components/BookForm/BookForm.tsx:112]: https://reactjs.org/docs/fragments.html#short-syntax

File: server/package.json (hunk 1 above)
Comment (manuatarchilogic) [reply server/package.json]: 0cc9eec

**Pull request merged**